diff -Naur EIGEN-original/cmake/EigenTesting.cmake EIGEN/cmake/EigenTesting.cmake
--- EIGEN-original/cmake/EigenTesting.cmake	2015-08-11 11:06:46.473190900 +0200
+++ EIGEN/cmake/EigenTesting.cmake	2015-08-11 12:40:37.737941800 +0200
@@ -324,7 +324,7 @@
     # on all other system we rely on ${CMAKE_CXX_COMPILER}
     # supporting a "--version" or "/version" flag
     
-    if(WIN32 AND NOT CYGWIN)
+    if(MSVC)
       set(EIGEN_CXX_FLAG_VERSION "/version")
     else()
       set(EIGEN_CXX_FLAG_VERSION "--version")
